concrete/src/Multilingual/Service/Extractor.php

```diff
@@ -22,9 +22,14 @@ public function extractTranslatableSiteStrings()
     {
         $translations = new Translations();
         $translations->insert('SiteName', \Core::make('site')->getSite()->getSiteName());
+        $fh = \Core::make('helper/file');
```

Comment:
Any chance you can do a quick update to using Application::getFacadeApplication() instead of \Core in this file?